extensions/aris/src/bigDataCluster/controller/controller.ts

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { EndpointRouterApi } from './apiGenerated';
+import { IEndPointsResponse, IControllerError, IEndPointsRequest, IHttpResponse, IEndPoint } from './types';
+
+export class ArisController {
```

Comment:
>ArisController [](start = 13, length = 14)

This only contains static methods - why not just remove the class completely? There doesn't appear to be a reason to encapsulate them in a class structure.

Reply:
This design will change anyway based on RestfulAPI in the near future. I will leave as it is for now.